site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1567 @@
+              <li><button class="dropdown-item"><span class="visually-hidden">Sort by</span>Cheapest</button></li>
+              <li><button class="dropdown-item"><span class="visually-hidden">Sort by</span>Most expensive</button></li>
+            </ul>
+          </div>
+
+          <div class="d-flex">
+            <button class="btn btn-secondary d-lg-none flex-shrink-1" style="width: 146px;" data-bs-toggle="offcanvas" data-bs-target="#filterOffcanvas" aria-controls="filterOffcanvas">
+              <svg class="me-1" width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                <use xlink:href="#filter"/>
+              </svg>
+              <span class="visually-hidden">Toggle</span>Filter
+            </button>
+
+            <form><div class="btn-group d-none d-md-flex ms-2" role="group">
+              <input type="radio" class="btn-check" name="displayOffers" id="gridDisplay" autocomplete="off" checked>
+              <label class="btn btn-icon btn-no-outline" for="gridDisplay" title="Grid">
+                <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                  <use xlink:href="#grid"/>
+                </svg>
+                <span class="visually-hidden">Grid display</span>
+              </label>
+              <input type="radio" class="btn-check" name="displayOffers" id="listDisplay" autocomplete="off">
+              <label class="btn btn-icon btn-no-outline" for="listDisplay" title="List">
+                <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                  <use xlink:href="#list"/>
+                </svg>
+                <span class="visually-hidden">List display</span>
+              </label>
+            </div></form>
+          </div>
+        </div>
+
+        <p class="pt-md-3 h4 mb-3">
+          <span class="text-primary">16 matches</span> / <span class="d-none d-xl-inline">9</span><span class="d-xl-none">6</span> displayed
+        </p>
+
+        <ul class="row row-cols-1 row-cols-md-2 row-cols-xl-3 gy-3 mb-3 list-unstyled">
+          <li class="card border-0 pt-2 pt-md-4">
+            <h3 class="visually-hidden">xxxxx xxxxxx xx xxx xxx</h3>
+            <div class="d-flex position-relative mb-2 align-items-end justify-content-between border-bottom border-primary fw-bold">
+              <div class="d-flex">
+                <div class="p-2 bg-primary text-nowrap">
+                  <svg class="me-1" width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                    <use xlink:href="#tick"/>
+                  </svg>Flash sale
+                </div>
+                <div class="custom-triangle border-primary"></div>
+              </div>
+              <div class="position-absolute end-0">
+                <p class="mb-1 small fw-bold lh-sm text-nowrap">85 items left!</p>
```

Comment:
Not sure if it will be easy enough for someone using a screen reader to understand that there is a timer for this flash sale **AND** that this offer will be valid while stocks last. May need to clarify this point.

Reply:
Waiting for a11y experts, we can keep this comment open until we reach an agreement.